crates/jstz_proto/src/executor/contract.rs

```diff
@@ -131,23 +134,14 @@ impl Script {
         Ok(Self(module))
     }
 
-    // TODO: we need to be able to specify the type of console API (Proto vs Cli),
-    // With current implementation, calling a contract in CLI will revert the logging back to Proto
     fn register_apis(
         &self,
         contract_address: Address,
-        context: &mut Context<'_>,
```

Comment:
What's the purpose of this change?

Reply:
i'm sticking to the conventions to place context at the end, similar to how we place runtime at the start :)